static/js/impala/ajaxcache.js

```diff
@@ -92,6 +92,9 @@ $.ajaxCache = function(o) {
             }
         }
     }
+    $("#live-announcements").html(gettext(
```

Comment:
The example yahoo page doesn't clear the string ... When do we need to clear the string ? a use case I mean